src/blog/2025-09-19-oss-summit-europe-2025.md

```diff
@@ -0,0 +1,101 @@
+
+Move beyond vanity metrics. Report progress against a simple journey that leadership already understands.
+
+* **Time to first value.** How long does it take a new developer to achieve their first success that matters; a successful build, a working webhook, or a deployed integration.  
+* **Activation.** The share of new developers who reach that first value within a defined period. For many products, that is within the first 7 or 14 days.  
+* **Adoption.** Depth of use after activation. Are developers using the next API, the next workflow, or the next feature that increases stickiness?  
+* **Retention.** Do activated developers return and keep building? Track it by cohort.  
+* **Influence on sales cycles.** Where DevRel shortens evaluation, reduces proof of concept time, or helps unblock procurement.
+
+Tie programs to these measures. Show the path from activity to outcome. For example, a tutorial refresh is not the outcome. The outcome is a lower time to first value for the target persona, confirmed by product analytics and user feedback. A community office hours session is not the outcome. The outcome is fewer stalled evaluations and more developers reaching activation in the same quarter.
+
+When you have outcomes, ask for resources to scale the programs that work. Use the business language your leadership uses. You are not asking for a headcount. You are proposing an investment that increases activation by a defined amount at a known cost.
+
+### How do we tie DevRel to revenue?
+
+Revenue is an outcome of successful journeys. DevRel can shorten and strengthen those journeys. The work is to trace developer progress across the community and the product.
+
+* **Connect the dots.** Use community intelligence and product analytics to see who is evaluating, who is unblocked, and who is converting. You do not need perfect attribution. You need a consistent way to see movement across key stages.  
+* **Instrument the handoffs.** Track how DevRel content and support reduce common blockers in proofs of concept. Track time from the first question to unblock. Track how often a successful proof of concept moves to a trial or purchase.  
+* **Capture stories and systematize them.** When you help a team move from evaluation to purchase faster, write the play. Which persona? Which blockers? Which content or support solved it? Turn that into a repeatable motion that anyone in your company can use.
+
+The Metrics working group is rebooting to publish examples you can adapt. Expect templates for funnel definitions, example dashboards, and sample reports that map DevRel outcomes to pipeline stages and customer health.
```

Comment:
not the working group

The [Metrics Index](https://github.com/DevRel-Foundation/wg-resource-aggregation?tab=readme-ov-file#working-group-projects) project aggregates metrics you can adopt.